src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.vbproj

```diff
@@ -281,9 +281,7 @@
       <DependentUpon>PackagePropPage.vb</DependentUpon>
       <SubType>UserControl</SubType>
     </Compile>
-    <Compile Update="PropPages\PackagePropPage.vb">
-      <SubType>UserControl</SubType>
```

Comment:
Do we know why we lost the `SubType`s?

Reply:
Shouldn't have, thanks for catching that. I'll fix in next iteration.